service/selectors/entities/channels.js

```diff
@@ -148,3 +149,22 @@ export const getAutocompleteChannelWithSections = createSelector(
         return channels;
     }
 );
+
+export const canManageChannelMembers = createSelector(
+    getCurrentChannel,
+    getCurrentChannelMembership,
+    getCurrentTeamMembership,
+    getUsers,
+    getCurrentUserId,
+    (channel, channelMembership, teamMembership, allUsers, currentUserId) => {
+        const user = allUsers[currentUserId];
+        const roles = `${channelMembership.roles} ${teamMembership.roles} ${user.roles}`;
```

Comment:
@enahum It might be possible to use the same plain underlying function in both selectors. We'd probably change the selector to `makeGetCurrentUserRoles`. We should explore that.